rimage/transform/pinhole_camera_parameters.go

```diff
@@ -277,9 +277,9 @@ func (params *PinholeCameraIntrinsics) GetCameraMatrix() *mat.Dense {
 }
 
 // intrinsics2DPtTo3DPt takes in a image coordinate and returns the 3D point using the camera's intrinsic matrix.
-func intrinsics2DPtTo3DPt(pt image.Point, d rimage.Depth, pci *PinholeCameraIntrinsics) (r3.Vector, error) {
+func intrinsics2DPtTo3DPt(pt image.Point, d rimage.Depth, pci *PinholeCameraIntrinsics) r3.Vector {
 	px, py, pz := pci.PixelToPoint(float64(pt.X), float64(pt.Y), float64(d))
-	return r3.Vector{px, py, pz}, nil
+	return r3.Vector{px, py, pz}
```

Comment:
This change can be bubbled up further but ultimately ends up changing a public interface so didn't want to make that change